great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -891,26 +839,6 @@ def get_datasource(self, datasource_name: str = "default") -> FluentDatasource:
         datasource._data_context = self
         return datasource
 
-    @public_api
-    def add_store(self, store_name: str, store_config: StoreConfigTypedDict) -> Store:
```

Comment:
Removing the ability to configure stores through the API until we think this through more - please let me know if you disagree